extensions/amp-story-interactive/0.1/amp-story-interactive-img.css

```diff
@@ -91,3 +95,52 @@
   color: white !important;
   font-weight: 700 !important;
 }
+
+.i-amphtml-story-interactive-active:not(.i-amphtml-story-interactive-post-selection)
+  .i-amphtml-story-interactive-img-option-img {
+  animation: i-amphtml-story-interactive-img-option-img-landing-animation
+    var(--i-amphtml-interactive-animation-time) !important;
+  animation-delay: var(
+    --i-amphtml-interactive-landing-animation-delay-offset-el1
+  ) !important;
+}
+
+.i-amphtml-story-interactive-active:not(.i-amphtml-story-interactive-post-selection)
+  .i-amphtml-story-interactive-img-option-img::before {
+  display: block !important;
+  background-color: var(--i-amphtml-interactive-theme-shading-flash) !important;
+  opacity: 0 !important;
+  animation: i-amphtml-interactive-animation-landing-animation-flash-background
+    var(--i-amphtml-interactive-animation-time) !important;
+  animation-delay: inherit !important;
+}
```

Comment:
Thank you for writing this. It's great to see it in the demo.
I agree with your hunch that it will look odd to have a color overlay on the image.

Let's remove this. The scaling is enough to bring attention to the element.

Reply:
I committed the changes and just updated the Firebase demo